spec/requests/provider_management/external_users_spec.rb

```diff
@@ -0,0 +1,156 @@
+require 'rails_helper'
+
+RSpec.describe 'providers external users management', type: :request do
+  describe 'GET /providers_management/providers/:provider_id/external_users/:id/disable' do
```

Comment:
Typo in test description.

```suggestion
  describe 'GET /provider_management/providers/:provider_id/external_users/:id/disable' do
```

Reply:
👍